src/main/scala/io/tardieu/netwemo/checkers/Checker.scala

```diff
@@ -4,13 +4,14 @@ import akka.actor.{Actor, ActorLogging}
 import io.tardieu.netwemo.connectors.{NetatmoConnector, WemoConnector}
 
 import scala.concurrent.Future
-import scala.concurrent.ExecutionContext.Implicits.global
 import scala.util.{Failure, Success}
 
 case object RunCheck
 
 trait Checker extends Actor with ActorLogging {
 
+  private[this] implicit val executionContext = context.system.dispatcher
```

Comment:
I seem to recall that `context.dispatcher` is available in recent Akka versions at least